migtests/tests/pg/assessment-report-test/init-db

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
+
+set -e
+set -x
+
+source ${SCRIPTS}/functions.sh
+
+
+echo "Creating ${SOURCE_DB_NAME} database on source"
+run_psql postgres "DROP DATABASE IF EXISTS ${SOURCE_DB_NAME};"
+run_psql postgres "CREATE DATABASE ${SOURCE_DB_NAME};"
+
+echo "Initialising source database."
+
+run_psql ${SOURCE_DB_NAME} "\i ${TEST_DIR}/../misc-objects-1/schema.sql;"
+run_psql ${SOURCE_DB_NAME} "\i ${TEST_DIR}/../misc-objects-2/pg_misc_objects2.sql;"
+run_psql ${SOURCE_DB_NAME} "\i ${TEST_DIR}/../views-and-rules/pg_views_and_rules_automation.sql;"
+run_psql ${SOURCE_DB_NAME} "\i pg_assessment_report.sql;"
```

Comment:
how about defining all of this also in a temporary SQL file and running that at once.